### DISCUSSION TITLE
Webrtc Playing With H265 support

### DISCUSSION THREAD

**Discussion by kalmastenitin** [bluenviron/mediamtx#2589] [Questions]
### Question

RTSP Server with H265 stream is not playable via webrtc. is there any specific reason its support not added? Are we planning to add h265 support to play over webrtc?

**Comment by aler9** [comment]
<s>Hello, H265 through WebRTC is not supported by any major browser and it will probably never be due to patents and strategic choices.

The server doesn't have anything to do with this limitation.

Open an issue in Firefox, Chrome and Safari trackers if you want to discuss further.</s>

**Comment by aler9** [comment]
HLS on the other hand supports H265 when there's a hardware decoder available, so you can use that to perform transcoding from RTSP to the browser.

**Comment by saket424** [comment]
@aler9 
Are there any plans to support h265 playback using the MSE specifications as implemented by the go2rtc project 

https://github.com/AlexxIT/go2rtc/tree/v1.9.4?tab=readme-ov-file#module-mp4

**Reply by aler9** [reply]
We already support reading H265 through LL-HLS, and it's way faster than MSE. If you still have questions, open a dedicated issue.

**Comment by aler9** [comment]
streaming H265 tracks with WebRTC has been implemented with #4003
